inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -40,10 +40,15 @@
 def experiment_names(self, key, value):
     """Experiment names."""
     if value.get('u'):
-        self.setdefault('affiliation', [])
-        raw_affiliations = force_force_list(value.get('u'))
-        for raw_affiliation in raw_affiliations:
-            self['affiliation'].append(raw_affiliation)
+        self.setdefault('affiliations', [])
+        name = value.get('u')
+        recid = value.get('z')
+        record = get_record_ref(recid, 'institutions')
+        self['affiliations'].append({
+            'curated_relation': record is not None,
+            'name': name,
+            'record': record
+            })
```

Comment:
Spacing is much better in https://github.com/inspirehep/inspire-next/pull/1451/files#diff-8f850f5dde975afda8a95a3023d5005cR124.